src/patternfly/components/DataList/data-list.scss

```diff
@@ -67,6 +67,14 @@
   --pf-c-data-list__cell--cell--PaddingTop: 0;
   --pf-c-data-list__cell--cell--md--PaddingTop: var(--pf-global--spacer--lg);
   --pf-c-data-list__cell--m-icon--cell--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-data-list__cell--MaxWidth: none;
+  --pf-c-data-list__cell--MinWidth: 0;
+  --pf-c-data-list__cell--Overflow: visible;
+  --pf-c-data-list__cell--TextOverflow: clip;
+  --pf-c-data-list__cell--WhiteSpace: normal;
+  --pf-c-data-list__cell--WordBreak: normal;
+  --pf-c-data-list__cell--m-truncate--MaxWidth: 100%;
+  --pf-c-data-list__cell--m-truncate--MinWidth: calc(20ch);
```

Comment:
no need for calc(), and this should be something much smaller. it's 5ch in the table

https://github.com/patternfly/patternfly/blob/1a1a1a84c6e1efaa65b1b872c0aa306755b868fb/src/patternfly/components/Table/table.scss#L63